package.json

```diff
@@ -8,17 +8,14 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.1.2",
-    "@storybook/addon-centered": "^5.0.6",
-    "@storybook/addon-options": "5.0.6",
-    "@storybook/addon-viewport": "5.0.6",
-    "@storybook/react": "5.0.6",
```

Comment:
after updating `styled-components` I got the following error 
`ERROR in ./node_modules/styled-components/dist/styled-components.browser.esm.js
Module not found: Error: Can't resolve 'react-is' in '/Users/yuyok/workspace/github/react-flow-chart/node_modules/styled-components/dist'` - updating `storybook` fixed the issue though.